webpack/stories/index.js

```diff
@@ -14,7 +14,7 @@ require('../assets/javascripts/bundle');
 require('../../app/assets/javascripts/application');
 require('../../app/assets/stylesheets/base.scss');
 
-require('patternfly/dist/js/patternfly');
+// require('patternfly/dist/js/patternfly');
```

Comment:
won't this break storybook?

Reply:
No, and I have no explanation why it was there from the beginning as we load patternfly source file by es6 imports and let the webpack storybook to bundle them into one big bundle.

Reply:
ah, thats easy, the application.js rails =require statements are not honored in storybook, so without it you dont have the relevant JS/CSS files...

Reply:
Mmm, so I have tested it and it works without `patternfly/dist/js/patternfly`.
Maybe the code that uses the patternfly dist moved to webpack?

IMO we should not load the `app/assets/javascript/application.js` and the `webpack/assets/bundle.js` into the storybook and I plan to face it when implementing `@theforeman/env`.

**There is another issue that causes an error when running the storybook, I am uploading a fix as vendor alpha.7**